### PR TITLE
Added contribution footer for github and slack links.

### LINKED ISSUE #154: Developer link not in footer
FILL OUT THE FORM BELOW OR THE ISSUE WILL BE AUTO-CLOSED

**Issue Type (check one)**

- [ ] Bug Report
- [x ] Feature Idea
- [ ] Documentation
- [ ] Technical Discussion

**Description (Please type a clear and concise description below)** 

When looking through the app it might be convenient to have a link for the repo's on the site footer for better user experience.

### PR DESCRIPTION
**Pull request by JRobertEdwards**
#### What does this PR do?
- Adds the contribution items of slack and github links to the footer of the site

#### Description of Task to be completed?
- Not sure on this one but I believe it only needs merging

#### How should this be manually tested?
- run the site locally and visually check the footer is OK. Run react tests with the test:all script.

#### What are the relevant Trello Card stories? [ Trello Card Item ](https://trello.com/b/8x3mo2V5/say-their-names-web)
- Getting board not found - This was from a feature idea raised by myself here: https://github.com/Say-Their-Name/say-their-names-web/issues/154

#### Pull Request Type (check one)
- [x] Feature
- [ ] Bug Fixes
- [ ] Chore

#### Developer Checklist
- [x] New code has been scanned for ESlint issues
- [x] I have tested my code locally
- [x] I made sure that there are no other pull requests that solve for this issue
- [x] My code follows the same structure within the guidelines

#### Screenshots (if appropriate)
image

### CODE REVIEW

**Review by Easybuoy — COMMENTED** [review]
Great job with this task @JRobertEdwards.

On the mobile view, the contribution links are not well aligned, could you check that. 

Screenshot 2020-08-16 at 23 10 33

**Comment by JRobertEdwards** [comment]
@Easybuoy thanks! I believe the formatting issue is being caused by the svg image sizes being differently sized to the other svgs used in the social media links since they're all using the same styles. I think resizing the contributor images to the same size as the social media will fix the issue.

**Comment by hybridcattt** [comment]
I've looked into it, and indeed it's the SVGs - the width is set to `220` in both new svgs. In other images the width corresponds to the drawn paths, while here its's much wider. Submitting a suggestion with fixes

File: src/assets/github.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<svg width="220" height="28" viewBox="0 0 220 28" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (hybridcattt) [inline src/assets/github.svg:1]:
```suggestion
<svg width="85" height="28" viewBox="0 0 85 28" fill="none" xmlns="http://www.w3.org/2000/svg">
```

File: src/assets/slack.svg [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<svg width="220" height="28" viewBox="0 0 220 28" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (hybridcattt) [inline src/assets/slack.svg:1]:
```suggestion
<svg width="75" height="28" viewBox="0 0 75 28" fill="none" xmlns="http://www.w3.org/2000/svg">
```

**Review by hybridcattt — COMMENTED** [review]
Fixed the svgs :)

**Comment by Easybuoy** [comment]
> Fixed the svgs :)

Thanks @hybridcattt

File: README.md [hunk 3]
```diff
@@ -45,6 +45,8 @@ REACT_APP_NEWS_FETCH_PROXY=https://thingproxy.freeboard.io/fetch
 REACT_APP_TWITTER_URL=https://twitter.com/SayTheirName_io
 REACT_APP_FACEBOOK_URL=https://www.facebook.com/Say-Their-Names-108926117523025/?modal=admin_todo_tour
 REACT_APP_INSTAGRAM_URL=https://www.instagram.com/saytheirname.io
+REACT_APP_GITHUB_URL=https://github.com/Say-Their-Name
+REACT_APP_SLACK_URL=https://app.slack.com/client/T014JL5B3SN/C014X92G997
```
Comment (hybridcattt) [inline README.md:49]: Shouldn't this be the invite link instead? That's what we do in the apps at least.

Comment (JRobertEdwards) [reply README.md:49]: I've only used the links here that are from the contribution guidelines page. If there's better ones that are more intro friendly then I'm happy to have them changed. Maybe the contributions docs can be updated to reflect that as well?

Comment (Easybuoy) [reply README.md:49]: I don't understand, is the link invalid?

You're talking about the slack link right?

Comment (hybridcattt) [reply README.md:49]: Yes about the slack link. This link is only accessible to those who are already a member of our slack org. 
To join the org, one needs to sign up to it first, either by being directly invited or by using a special invite link. 
In the mobile apps we use the invite link on the about page, and it's shortened: https://bit.ly/say-their-names-slack. We use the same link in the contribution guidelines on GitHub.
This link is temporarily out of order, but we're working on getting it to work again 🤞

Comment (Easybuoy) [reply README.md:49]: Okay, since it's in environment variables it can be changed when it's sorted.

**Review by Easybuoy — APPROVED** [review]
Looks good

File: README.md (hunk 3 above)
Comment (hybridcattt) [reply README.md:49]: Indeed! It was more of an observation, I don't think it's necessary to change here. Sorry if that wasn't clear 😊

**Review by hybridcattt — APPROVED** [review]

**Pull request merged**